docs/robotics/motion/step_planning.md

```diff
@@ -0,0 +1,3 @@
+!!! warning
+
+    This is subject to change, as there are currently plans to not only plan an individual step, but a whole step sequence, which could increase speed when planning arond obstacles or similar.
```

Comment:
```suggestion
    This is subject to change, as there are currently plans to not only plan an individual step, but a whole step sequence, which could increase speed when planning around obstacles or similar.
```

Reply:
Did you forget to write something here or is \<nothing> subject to change? 😅

Reply:
😂 The step planning is subect to change. But okay, I will add some more information later, how it is currently done